Terminal.Gui/ConsoleDrivers/CursesDriver/UnmanagedLibrary.cs

```diff
@@ -256,7 +256,7 @@ static class Mono {
 		/// to avoid the dependency on libc-dev Linux.
 		/// </summary>
 		static class CoreCLR {
-#if NET7_0
+#if NET7_0 || NET8_0
```

Comment:
I think you want `NET7_0_OR_GREATER` instead.

Although conditional compilation only matters if the consumer builds it from source.

Reply:
Oh hey. Also...

Even though it's v1, this is probably something that should be updated at some point to use LibraryImport, and conditional compilation is nice to be more visible, such as including or excluding things at the project level, or similar.

But also.... Several things that class does are in netstandard2.0 and thus can just use the built-in methods so this doesn't remain a maintenance burden, like the IsWindows and such. 🤷‍♂️ 

And parts that aren't supported in netstandard2.0 but exist in more recent .net can be polyfilled by various source generators out there on nuget, and then platform-specific parts can use more standard facilities from that. The Meziantou.Polyfill is excellent for that and only emits what's actually used, at compile-time.

Reply:
`LibraryImport` is not supported in all project targets, so I think it is not a good alternative.

![image](https://github.com/gui-cs/Terminal.Gui/assets/13117724/c712691b-6eee-47f1-a573-0c8ca219fdef)

Reply:
I will use `NET6_0_OR_GREATER` to support `net6.0` which is still currently supported (LTS).